invenio_vocabularies/config.py

```diff
@@ -155,3 +155,14 @@
     "sort": ["name", "count"],
 }
 """Vocabulary type search configuration."""
+
+VOCABULARIES_ORCID_ACCESS_KEY = "TODO"
```

Comment:
Shouldn't this config be under "contrib"?

Reply:
That would imply to change how we initialize the config, see here: https://github.com/inveniosoftware/invenio-vocabularies/blob/master/invenio_vocabularies/ext.py#L70-L74

It's only getting the main config. To me the different contrib configs look more like static configs, not meant to be used/overriden at the app level.